tasks/bootstrap.yaml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Create ConfigMap yml from jinja template
+  template:
+    src: users_configMap.j2
+    dest: /tmp/users_configMap.yml
+
+- name: Read definition file and update auth-aws ConfigMap
+  k8s:
+     state: present
+     kubeconfig: /home/weslley/.kube/config
```

Comment:
I've to find a better solution for this. I don't know why I've to specify it once its is already the default path.

I know that it breaks if I don't specify, its breaks even if I specify what they suggest in the documentation:  https://docs.ansible.com/ansible/latest/modules/k8s_module.html